lib/ExpensiMark.js

```diff
@@ -93,11 +93,13 @@ export default class ExpensiMark {
             /**
              * Automatically links emails that are not in a link. Runs before the autolinker as it will not link an
              * email that is in a link
+             * Prevent emails from starting with [~_*] (such emails should not be supported), which is the reliable
+             * way to strip any enclosing ~_* in multi-line emails and in general.
```

Comment:
This comment is a little unclear to me, maybe something like: 

```
Prevent emails from starting with [~_*]. Such emails are not supported because we use [~_*] to format text.
```

Reply:
Oh "should not be supported" references the decision on the issue rather than that it happens to be implemented that way. So maybe:
```
Prevent emails from starting with [~_*]. Such emails should not be supported.
```

Reply:
Sure, that works too